.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,60 @@
+name: build-pr
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
+jobs:
+  build-pr:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get Composer Cache Directory
+      id: composer-cache
+      run: |
+        echo "::set-output name=dir::$(composer config cache-files-dir)"
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-composer-
+    - name: Composer Install
+      uses: php-actions/composer@v1
+      with:
+        args: install
+    - name: Setup Node.js 10
+      uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Cache Node - npm
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-cache-
+    - name: npm/Gulp Install
+      run: |
+        npm install
+        npm install gulp-cli -g
+    - name: Build develop version
+      run: |
+        npm run build:dev
```

Comment:
Not really scope of this PR, but the scripts in `package.json` are a bit inconsistent between how they work for production vs development. E.g. `build:dev` includes a call to `build:static` while `build:production` and `build:test` don't.

I'll open a small separate issue with this.